src/Http/OpenApi/src/SchemaGenerator.cs

```diff
@@ -0,0 +1,44 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.OpenApi;
+
+internal static class SchemaGenerator
```

Comment:
Curious: in what sense is this a "schema generator"❔ It seems limited to returning a `string` categorizing a `Type`

Reply:
It's a reductive implementation for something we plan on improving in an upcoming preview (see https://github.com/dotnet/aspnetcore/issues/41246).

A lot of the work for mapping .NET types over to JSON validation schemas is done in Swashbuckle's Swagger generator at the moment. Since we're adopting first-class support for OpenApi in this package, we want to roll out our own schema generator built-in.